modules/odc-interacting-with-applications-and-components.adoc

```diff
@@ -18,3 +18,9 @@ This feature is available only when you create applications using the *From Git*
 * Hover your cursor over the lower left icon on the Pod to see the name of the latest build and its status. The status of the application build is indicated as *New* (image:odc_build_new.png[title="New Build"]), *Pending* (image:odc_build_pending.png[title="Pending Build"]), *Running* (image:odc_build_running.png[title="Running Build"]), *Completed* (image:odc_build_completed.png[title="Completed Build"]), *Failed* (image:odc_build_failed.png[title="Failed Build"]), and *Canceled* (image:odc_build_canceled.png[title="Canceled Build"]).
 * Use the *Shortcuts* menu listed on the upper-right of the screen to navigate components in the *Topology* view.
 * Use the *List View* icon to see a list of all your applications and use the *Topology View* icon to switch back to the *Topology* view.
+* Use the *Display* dropdown menu to manipulate the various application groupings. These are automatically available depending on your project structure:
+** *Application Groupings*: Uncheck to disable application groupings and condense the application group into a card with an overview of an application group and alerts.
+** *Knative services*
```

Comment:
sentence for each, describe more in detail

Reply:
Do mention if the option shows/expands the details, based on the subsection it is located in.

Reply:
don't follow. @Preeticp